apps/dashboard/web/polaris_web/web/src/apps/dashboard/pages/quick_start/components/GithubSource.jsx

```diff
@@ -0,0 +1,150 @@
+import { HorizontalStack, LegacyCard, Text, TextField, VerticalStack } from "@shopify/polaris"
```

Comment:
abstract out the component, it is same as bitbucket source, also copy button will give error as ref is not defined, same bitbucket.jsx

Reply:
ye karte h